### PR TITLE
Redirects

### PR DESCRIPTION
**Pull request by escapemanuele**
I've been working on the dowload-images today, but in the meantime you could take a look at the redirects.
It may need some fixes, but take a look :)

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -7,21 +7,22 @@ require("dotenv").config({
 
 module.exports = {
   plugins: [
+    "gatsby-plugin-slug",
+    "gatsby-plugin-zeit-now",
```
Comment (Swizec) [inline gatsby-config.js:11]: That's cool, had no idea this existed!

File: gatsby-node.js [hunk 2]
```diff
@@ -1,9 +1,71 @@
...
+            // iterate through all `from` array items
+            from.forEach((from) => {
+              createRedirect({
+                fromPath: from,
+                toPath: to,
+                isPermanent: true,
+                redirectInBrowser: true
+              })
+            })
+          })
```
Comment (Swizec) [inline gatsby-node.js:63]: How come these can't be combined into the same loop?

File: gatsby-node.js [hunk 3]
```diff
@@ -1,9 +1,71 @@
...
+const chalk = require('chalk');
 
+const sharp = require("sharp")
 sharp.cache(false)
-sharp.simd(false)
\ No newline at end of file
+sharp.simd(false)
+
+exports.createPages = ({ graphql, actions }) => {
+    const { createRedirect } = actions
```
Comment (Swizec) [inline gatsby-node.js:8]: Added by that zeit plugin?

File: gatsby-node.js (hunk 2 above)
Comment (escapemanuele) [reply gatsby-node.js:63]: Damn, I should sleep more. Of course it can be combined

File: gatsby-node.js (hunk 3 above)
Comment (escapemanuele) [reply gatsby-node.js:8]: No, that's for generating client-side redirects. 'gatsby-plugin-zeit-now' is used for the server-side

**Pull request merged**